docs/content/manual/pre-release/resiliency/timeout.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Test timeout on loss of network connectivity"
+---
+
+## Ping Timeout
+
+1. Create a docker network:
+
+```shell
+docker network create -d bridge --subnet 192.168.22.0/24 longhorn-network
+```
+
+2. Start a replica:
+
+```shell
+docker run --net longhorn-network --ip 192.168.22.2 \
+         -v /volume longhornio/longhorn-engine:<tag> \
+         longhorn replica --listen 192.168.22.2:9502 --size 10g /volume
+```
+
+3. Start another replica:
+
+```shell
+docker run --net longhorn-network --ip 192.168.22.3 \
+         -v /volume longhornio/longhorn-engine:<tag> \
+         longhorn replica --listen 192.168.22.3:9502 --size 10g /volume
+```
+
+4. In another terminal, start the controller:
+
+```shell
+docker run --net longhorn-network --ip 192.168.22.4 --privileged \
+         -v /dev:/dev -v /proc:/host/proc \
+         longhornio/longhorn-engine:<tag> \
+         longhorn controller --replica tcp://192.168.22.2:9502 \
+         --replica tcp://192.168.22.3:9502 \
+         --frontend tgt-blockdev timeout-test
```

Comment:
We shouldn't use `tgt-blockdev` frontend since the logic has been moved into the instance manager.

Reply:
I'm not using the instance manager for this test.  I will wait until the instance manager being removed from longhornio/longhorn-engine is resolved to resolve this.

Reply:
Sounds good